src/pdf_beamtime/src/pdf_beamtime_server.cpp

```diff
@@ -81,22 +84,63 @@ rclcpp_action::CancelResponse PdfBeamtimeServer::handle_cancel(
 void PdfBeamtimeServer::execute(
   const std::shared_ptr<rclcpp_action::ServerGoalHandle<PickPlaceControlMsg>> goal_handle)
 {
-  // A cool Moveit function
-  std::vector<double> home_angles = node_->get_parameter("home_angles").as_double_array();
+  const auto goal = goal_handle->get_goal();
+  //  Variables for feedback and results
+  auto feedback = std::make_shared<PickPlaceControlMsg::Feedback>();
+  auto results = std::make_shared<PickPlaceControlMsg::Result>();
+  bool fsm_results = true;
+  auto goal_home = node_->get_parameter("home_angles").as_double_array();
+  feedback->status = get_state_completions();
+
+  RCLCPP_INFO(node_->get_logger(), "Current state is state %d", static_cast<int>(current_state_));
```

Comment:
Add some of the below logic to this RCLCPP_INFO message. 
"We're at this state, because of that, we are doing this first then this, or just doing the thing"

Reply:
RCLCPP_INFO text was changed.
https://github.com/maffettone/erobs/blob/a06a1507719f619a990c28445e33045aeb015100/src/pdf_beamtime/src/pdf_beamtime_server.cpp#L97